data/design/agi_designers.json

```diff
@@ -0,0 +1,715 @@
+[
+  "Aage Sikker Hansen",
+  "Yung-Chen Nieh",
+  "Adrian Frutiger",
+  "Adrian Shaughnessy",
+  "Aimo Katajam\u00e4ki",
```

Comment:
If the file is UTF-8 encoded, this could be "Aimo Katajamäki" rather than escaping characters. (Same below for others with `\uXXXX` values.)